app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html

```diff
@@ -30,15 +30,15 @@
 @analyticsAdditionalJs = {
     ga('set', {
         'dimension7':  '@spSummary.contextMessage.getOrElse("")',
-        'dimension8':  '@spSummary.forecastAmount.week.toInt',
+        'dimension8':  '@spSummary.forecast.forecastAmount.week.toInt',
         'dimension10': '@spSummary.numberOfQualifyingYears',
         'dimension11': '@spSummary.numberOfGaps',
         'dimension12': '@spSummary.numberOfGapsPayable',
         'dimension13': '@spSummary.yearsToContributeUntilPensionAge',
         'dimension14': '@spSummary.contractedOutFlag',
         'dimension15': '@spSummary.statePensionAge',
         'dimension16': '@spSummary.copeAmount.week',
-        'dimension21': '@spSummary.forecastOnlyFlag',
+        'dimension21': '@spSummary.forecast.scenario',
```

Comment:
We agreed with Paul to replace the "contextMessage" with scenario. So remove it to dimension7, we should just get rid of dimension21